packages/js/product-editor/src/utils/register-product-editor-block-type.ts

```diff
@@ -15,12 +14,17 @@ interface BlockRepresentation< T extends Record< string, object > > {
 	settings: Partial< BlockConfiguration< T > >;
 }
 
+// Define a more generic type for the select function to avoid TypeScript errors
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
With the `any` type removed, we no longer need this comment.

Reply:
Removed in 6b5aa8b.